data/eos-demo-video.desktop.in.in

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+_Name=Demo Video
+Exec=@LIBEXECDIR@/eos-demo-video
+Terminal=false
+Categories=
+Type=Application
+NoDisplay=true
+AutostartCondition=if-exists /home/demo-guest
```

Comment:
> but a slight corner case is that unless you remove the file after login, it would probably check for "is the demo guest user logged in" instead. (I think it'd still be OK.)

I'm not sure if this is a problem. Apps in `xdg/autostart` are only run when the session manager starts up, right?